heroku_deploy/entrypoint.sh

```diff
@@ -69,8 +69,19 @@ main(){
     app_name=${INPUT_HEROKU_APP_NAME_STAGING_THREE}
 
   fi
-  
-  git push https://heroku:${INPUT_HEROKU_API_KEY}@git.heroku.com/${app_name}.git HEAD:master -f
+
+  # The preboot will be always active, only when the label “migration” is present will be disabled.
+  # TODO: Once tested, 'staging_3' must be chenged bye 'production'
```

Comment:
```suggestion
  # TODO: Once tested, 'staging_3' must be changed to 'production'
```
for a brief moment, the 'bye production' thing scared the shit out of me

Reply:
OMG, this is embarrassing 🤣.
Thank you